docs/5. Integrations/Speech to text capabilities in Glific.md

```
@@ -0,0 +1,95 @@
+
+
+By specifying the `output_language` parameter, this configuration ensures that all voice notes are transcribed and the final text is available consistently in the same language, no matter the language of the incoming voice note. 
+
+
+## Using other providers 
+Apart from Gemini which is the default speech engine used by Glific. Eleven Labs can be used as a provider. This can be configured as the provider to use for transcription purposes be explicitly providing additional parameters of `provider` and `model` 
+
+### Eleven Labs
+
+To reference the eleven labs provider, pass the following additional parameters in the webhook body
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)